web/src/personal_menu_popover.ts

```diff
@@ -105,6 +106,36 @@ export function initialize(): void {
                 popovers.hide_all();
                 e.preventDefault();
             });
+
+            $popper.on("click", ".info-density-controls button", function (this: HTMLElement, e) {
+                const changed_property =
+                    information_density.information_density_properties_schema.parse(
+                        $(this).parent().attr("data-property"),
+                    );
+                const new_setting_value =
+                    information_density.get_new_value_for_information_density_settings(
+                        $(this),
+                        changed_property,
+                    );
+                const data = information_density.update_information_density_settings(
+                    $(this),
+                    changed_property,
+                    new_setting_value,
+                );
+                information_density.enable_or_disable_control_buttons($popper);
+                void channel.patch({
+                    url: "/json/settings",
+                    data,
+                });
+                e.preventDefault();
+            });
+
+            const resizeObserver = new ResizeObserver(() => {
+                void instance.popperInstance?.update();
+            });
+            resizeObserver.observe(document.querySelector("#personal-menu-dropdown")!);
```

Comment:
Probably we want to request an animation frame a la 196f4e92a3c4e722cdf14379eba3190b1aafdfaa, to avoid that potential failure mode.

Reply:
Done.